### PR TITLE
update to 1.20.6 (& small gradle fixes)

### LINKED ISSUE #16: 1.20.6 Support?
Hello, is this mod discontinued or just forgot to update? If development didn't stopped, can you update this mod to 1.20.6 update?

Thank you as always SkywardMC team!

### PR DESCRIPTION
**Pull request by uku3lig**
this pr simply updates the mod to 1.20.6 (despite no changes being needed), and fixes a few small things in the gradle buildscript, see 4590c986b427ee7bf304958cc1669b2fc2893550

hopefully i haven't forgotten anything :)

fixes #16

### CODE REVIEW

**Review by intergrav — APPROVED** [review]
lgtm

File: build.gradle [hunk 1]
```diff
@@ -1,73 +1,43 @@
...
-processResources {
-	inputs.property "version", project.version
+    // Fabric API. This is technically optional, but you probably want it anyway.
+    modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
 
-	filesMatching("fabric.mod.json") {
-		expand "version": project.version
-	}
+    modImplementation "com.terraformersmc:modmenu:${project.mod_menu_version}"
 }
```
Comment (LostLuma) [inline build.gradle:22]: You removed the `processResources` block even though the mod version needs to be replaced in the `fabric.mod.json` file still.
```suggestion
    // Fabric API. This is technically optional, but you probably want it anyway.
    modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"

    modImplementation "com.terraformersmc:modmenu:${project.mod_menu_version}"
}

processResources {
	inputs.property "version", project.version

	filesMatching("fabric.mod.json") {
		expand "version": project.version
	}
}
```
Not sure if my suggested change is completely right, the editor on here is kinda bad ...

Comment (uku3lig) [reply build.gradle]: argh, i don't know how i accidentally removed that and didn't notice it...

thank you !

**Review by getchoo — APPROVED** [review]
tysm!

**Pull request merged**